### DISCUSSION TITLE
why not to be consistent with colorScheme property ?

### DISCUSSION THREAD

**Discussion by AlonGvili** [chakra-ui/chakra-ui#3992] [General] [answered]
I can't understand why some components come with built-in colorScheme property, and some need to implement ourselves?

thanks in advance

**Comment by chr-ge** [comment, marked as answer]
- `color` is used to give text a color
- `colorScheme` prop represents a visual color scheme, not a css color attribute.
Read more about it here: #2846 (comment), #2846 (reply in thread)

**Reply by AlonGvili** [reply]
> * `color` is used to give text a color
> * `colorScheme` prop represents a visual color scheme, not a css color attribute.
>   Read more about it here: #2846 (comment), #2846 (reply in thread)

I understand that, but it's not what i asked.

for example the Radio component have built-in support for the colorScheme, so you can select a color from the scheme and it reflects on the radio when his status is checked.

but for  the Text component => Color Schemes for Text are not implemented in the default theme
and i asked why not ??

**Reply by chr-ge** [reply]
The `colorScheme` behind the scenes uses the different palette ranges for its color and `hover`, `disabled`, `active` pseudo-classes. 
For example, if you set the `colorScheme` of a button = 'blue', it will set the button bgColor to **blue.500**, the _hover bgColor to **blue.600**, etc 
Text does not have these different states, its only just the color of the text.

**Reply by AlonGvili** [reply]
Thanks, so basically component that have those states should have colorScheme by default.

**Reply by chr-ge** [reply]
Yes, to customize the color of components that have "states" you need `colorScheme`.

**Reply by AlonGvili** [reply]
> Yes, to customize the color of components that have "states" you need `colorScheme`.

maybe add this as a note in the docs.
